control-plane-operator/controllers/hostedcontrolplane/ignition/reconcile.go

```diff
@@ -89,7 +95,23 @@ func SetMachineConfigLabels(mc *mcfgv1.MachineConfig) {
 	}
 }
 
-func reconcileImageContentPolicyIgnitionConfigMap(cm *corev1.ConfigMap, imageDigestMirrorSet *configv1.ImageDigestMirrorSet, ownerRef config.OwnerRef) error {
+func reconcileImageContentSourcePolicyIgnitionConfigMap(cm *corev1.ConfigMap, icsp *v1alpha1.ImageContentSourcePolicy, ownerRef config.OwnerRef) error {
```

Comment:
You could use the same code as below by just changing the parameter type from *v1alpha1.ImageContentSourcePolicy or *configv1.ImageDigestMirrorSet to client.Object